complement/pipeline.yml

```diff
@@ -37,3 +40,60 @@ steps:
       automatic:
         - exit_status: 128
           limit: 3
+
+  - command:
+      # Download the latest checkout of Synapse develop
+      - "wget -N https://github.com/matrix-org/synapse/archive/develop.tar.gz -O synapse.tar.gz && tar -xzf synapse.tar.gz && cd synapse"
+      # Build the Synapse production docker image
+      - docker build -t matrixdotorg/synapse -f docker/Dockerfile .
```

Comment:
We really have to do this and can't use the docker hub version?

Reply:
Now that we're using the latest Synapse release we can totally use dockerhub!